### PR TITLE
How-To guide: base Kubernetes YAML

### PR DESCRIPTION
**Pull request by willtsai**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request: 

## Description

Taking over https://github.com/radius-project/docs/pull/748 from @jasonviviano to add a how-to guide on applying a base Kubernetes YAML using Radius.

## Issue reference

Fixes: #703

### CODE REVIEW

File: docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## Step 3: Deploy the application and containers
+
+Ensure that your `manifest.yaml` and `app.bicep` files are in the same directory, then deploy your application:
+
+```bash
+rad deploy app.bicep
+```
+
+> **Note**: in the deployment, Radius will override any parameters in the base Kubernetes YAML manifest (i.e. `manifest.yaml` file) with any values specified in the Radius application definition (i.e. `app.bicep` file) for the same parameters.
```
Comment (willtsai) [inline docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md:44]: @youngbupark - please review this for accuracy

Comment (youngbupark) [reply docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md:44]: Yes it is right. Can you please add the `Limitation` section which describe supported kubernetes resources ?

You can find the details in the following code. 

https://github.com/radius-project/radius/blob/c262df7aee22cf1ebea2fa66d3e70cfdaa135ca0/pkg/corerp/frontend/controller/containers/validator.go#L114-L119

Comment (willtsai) [reply docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md:44]: @youngbupark 

Thanks, the limitations are already live in the docs: https://docs.radapp.io/guides/author-apps/containers/overview/#base-kubernetes-yaml

I also included a reference to it in this page: https://github.com/radius-project/docs/pull/1024/files#diff-2db3e8ae3a31ec4507e63e2e0b2453d1d2f0b6dc2205502e1487e964b925ef54R34

**Comment by willtsai** [comment]
consider using this example instead to illustrate this feature: https://github.com/willdavsmith/radius/blob/willdavsmith/0.30-release-fixes/test/functional/shared/resources/testdata/manifest/basemanifest.yaml

**Pull request closed without merging**